src/py21cmfast/wrapper/cfuncs.py

```diff
@@ -370,6 +370,13 @@ def construct_fftw_wisdoms(
 #  They are mostly used for testing but may be useful in some post-processing applications
 
 
+def get_delta_crit(user_params, cosmo_params, mass, redshift):
+    """Gets the critical collapse density given a mass, redshift and parameters."""
+    sigma, _ = evaluate_sigma(user_params, cosmo_params, mass)
+    growth = lib.dicke(redshift)
+    return np.vectorize(lib.get_delta_crit)(user_params.cdict["HMF"], sigma, growth)
```

Comment:
It might be useful to have a version of `get_delta_crit` to which you just pass `sigma` directly (instead of mass).